recipe/meta.yaml

```diff
@@ -28,12 +29,13 @@ source:
    # library paths, even if BUILD_SHARED_LIBS=1
    # This patch fixes that.
     - 0003-LEMONConfig.cmake.patch
+   # export symbols for shared lib on windows
+    - 0004-allow-symbols-to-be-exported-in-shard-libs-on-win.patch  # [win]
 
 build:
-  number: 2
+  number: 10
```

Comment:
```suggestion
  number: 3
```

Reply:
> Small note: I set the build number to 10 to make it more easily distinguishable from previous static builds.

so you're not a fan? :+1:

Reply:
right, yeah, just keep it simple. it will get reset to on then next update anyway

Reply:
i don't think we can realistically expect a version bump in lemon :D